R/metrics.R

```diff
@@ -264,7 +264,7 @@ as_tibble.metric_set <- function(x, ...) {
   metrics <- unname(metrics)
 
   classes <- map_chr(metrics, class1)
-  directions <- map_chr(metrics, get_metric_fn_direction)
+  directions <- map_chr(metrics, get_fn_direction)
```

Comment:
Maybe keep this one as `get_metric_fn_direction()` because I do feel that adds some value

Reply:
https://github.com/tidymodels/yardstick/pull/322/commits/673f98f13eb3903b35c6ff3e6aa9226cc17b207d